Cargo.toml

```diff
@@ -10,20 +10,22 @@ readme = "README.md"
 keywords = ["ecs", "entity"]
 categories = ["data-structures", "game-engines"]
 
-[badges]
-maintenance = { status = "actively-developed" }
-
```

Comment:
Why was this moved?

Reply:
I was lazy and used `cargo add hashbrown`, and somewhere in the reading/writing process it decided to also move this.